sympy/core/evalf.py

```diff
@@ -1439,7 +1439,11 @@ def evalf(self, n=15, subs=None, maxn=100, chop=False, strict=False, quad=None,
 
         chop : bool or number, optional
             Specifies how to replace tiny real or imaginary parts in
-            subresults by exact zeros.
+            subresults by exact zeros. e.g.
+            ``a = Rational('37751345442791/1000000000000000000000000')
+            b = exp(-24)
+            (a-b).evalf(n=20, chop=True)  #incorrect method
+            a.expand().evalf(n=20, chop=True) - b.expand().evalf(n=20, chop=True) #correct method``
```

Comment:
I don't think this would be the right way to update docstrings. A better way could be to add these two examples  with comments showing the difference between the output of the two statements , rather than stating one incorrect.

https://github.com/sympy/sympy/blob/f9badb21b01f4f52ce4d545d071086ee650cd282/sympy/core/evalf.py#L1449-L14456

Adding two statements like this with desired output would be much better.


But still this may be a temporary solution, since final decision of how `chop` shall behave is yet to be taken.

Reply:
@sidhu1012 Why these checks are failing